### PR TITLE
add new rule logind_session_timeout

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- add rule which configures Logind to terminate idle sessoins
- add OVAL, Bash, Ansible and test scenarios
- the rule uses XCCDF variable
- one change needed to be done to a Bash ini_file macro

#### Rationale:

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=2122322

### CODE REVIEW

**Comment by ggbecker** [comment]
I believe you git tree is a bit messed up. Try rebasing it one more time.

**Comment by vojtapolasek** [comment]
Thanks for noticing that, I rebased.

File: linux_os/guide/system/accounts/accounts-physical/logind_session_timeout/oval/shared.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  </unix:file_object>
+
+  <ind:textfilecontent54_state id="state_logind_session_timeout" version="1">
+    <ind:subexpression datatype="int" operation="equals" var_ref="var_logind_session_timeout" />
+  </ind:textfilecontent54_state>
+
+  <external_variable id="var_logind_session_timeout" datatype="int"
+  comment="idle session timeout in seconds" version="1" />
+
+</def-group>
```
Comment (marcusburghardt) [inline linux_os/guide/system/accounts/accounts-physical/logind_session_timeout/oval/shared.xml:36]: Empty line is missing at the end. : )

Comment (vojtapolasek) [reply linux_os/guide/system/accounts/accounts-physical/logind_session_timeout/oval/shared.xml]: I fixed it by rebase.

**Comment by Mab879** [comment]
Looks like something is off on the rebase fa7fcd02fcb8e07d1b8ba713f4c85c9f0b10b8e7 looks odd to me as you are commiting a change made by me.

**Comment by jan-cerny** [comment]
@vojtapolasek can you try to take a look whether the rebase is correct or can you rebase again?

**Comment by jan-cerny** [comment]
@vojtapolasek bump

**Comment by vojtapolasek** [comment]
@jan-cerny  rebased

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**